MaxMind.MinFraud/Request/Event.cs

```diff
@@ -71,35 +71,36 @@ public sealed class Event
             TransactionId = transactionId;
             ShopId = shopId;
             Time = time;
-            this.Type = type;
+            Type = type;
         }
 
         /// <summary>
         /// Your internal ID for the transaction. We can use this to locate a
         /// specific transaction in our logs, and it will also show up in email
         /// alerts and notifications from us to you.
         /// </summary>
-        [JsonProperty("transaction_id")]
+        [JsonPropertyName("transaction_id")]
         public string? TransactionId { get; }
 
         /// <summary>
         /// Your internal ID for the shop, affiliate, or merchant this order is
         /// coming from. Required for minFraud users who are resellers, payment
         /// providers, gateways and affiliate networks.
         /// </summary>
-        [JsonProperty("shop_id")]
+        [JsonPropertyName("shop_id")]
         public string? ShopId { get; }
 
         /// <summary>
         /// The date and time the event occurred.
         /// </summary>
-        [JsonProperty("time")]
+        [JsonPropertyName("time")]
         public DateTimeOffset? Time { get; }
 
         /// <summary>
         /// The type of event being scored.
         /// </summary>
-        [JsonProperty("type")]
+        [JsonConverter(typeof(EnumMemberValueConverter<EventType>))]
```

Comment:
Do one of our tests cover serializing this using this?

Reply:
Yeah, we do test the serialization of the request sent to the mock web service. However, a direct test sounds worthwhile. I'll add that.